src/utils/loginUtils.js

```diff
@@ -31,9 +31,15 @@ const createAccount = async (userId, accountInfo, role) => {
 //if session and userInfo exist, return them
 //else return null and the user will have to sign in again
 const getSession = async () => {
-  const session = JSON.parse(window.localStorage.getItem('session'))
-  const userInfo = JSON.parse(window.localStorage.getItem('userInfo'))
-  if (session && userInfo) {
+  const session = JSON.parse(window.localStorage.getItem('session'));
```

Comment:
Maybe create a function for JSON.parse(window.localStorage.getItem() so you don't have to repeat it 4 times. It may or may not save lines, but I remember doing it for FEC